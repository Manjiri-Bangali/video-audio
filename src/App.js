import './App.css';
import Signup from './components/signup/signup';
import Signin from './components/signin/signin';
import { Routes, Route } from "react-router-dom";

function App() {
  return (
    <div className="App">
      <Routes>
        <Route path="/signup" element={<Signup />} />
        <Route path="/" element={<Signin />}/>
      </Routes>
    </div>
  );
}

export default App;
