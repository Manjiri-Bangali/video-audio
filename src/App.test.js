import { render, screen } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import App from './App';

describe('App.js', ()=>{
  it('should render the component', () => {
    const ele = render(<BrowserRouter><App/></BrowserRouter>);
    expect(ele).toMatchSnapshot();
  });
});
