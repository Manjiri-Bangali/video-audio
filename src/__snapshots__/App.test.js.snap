// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.js should render the component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="App"
      >
        <div
          class="login-div signin-component"
        >
          <div
            class="shapes yellow-circle"
          />
          <div
            class="shapes green-circle"
          />
          <div
            class="shapes red-circle"
          />
          <h3>
            Sign In
          </h3>
          <div
            class="content"
          >
            <form>
              <div>
                <label
                  class="input-label"
                  for="email"
                >
                  Email address
                </label>
                <input
                  id="email"
                  name="email"
                  placeholder="Enter your Email address"
                  type="text"
                  value=""
                />
              </div>
              <div
                class="panel"
              >
                <label
                  class="input-label"
                  for="password"
                >
                  Password
                </label>
                <a
                  class="forgot-label"
                  href="/signup"
                >
                  Forgot?
                </a>
                <input
                  id="password"
                  name="password"
                  placeholder="Enter your password"
                  type="password"
                  value=""
                />
              </div>
              <button
                class="submit-btn"
                type="submit"
              >
                Login
              </button>
              <div
                class="panel signup-panel"
              >
                <span>
                  New here? 
                </span>
                <a
                  class="forgot-label"
                  href="/signup"
                >
                  Signup
                </a>
              </div>
            </form>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="App"
    >
      <div
        class="login-div signin-component"
      >
        <div
          class="shapes yellow-circle"
        />
        <div
          class="shapes green-circle"
        />
        <div
          class="shapes red-circle"
        />
        <h3>
          Sign In
        </h3>
        <div
          class="content"
        >
          <form>
            <div>
              <label
                class="input-label"
                for="email"
              >
                Email address
              </label>
              <input
                id="email"
                name="email"
                placeholder="Enter your Email address"
                type="text"
                value=""
              />
            </div>
            <div
              class="panel"
            >
              <label
                class="input-label"
                for="password"
              >
                Password
              </label>
              <a
                class="forgot-label"
                href="/signup"
              >
                Forgot?
              </a>
              <input
                id="password"
                name="password"
                placeholder="Enter your password"
                type="password"
                value=""
              />
            </div>
            <button
              class="submit-btn"
              type="submit"
            >
              Login
            </button>
            <div
              class="panel signup-panel"
            >
              <span>
                New here? 
              </span>
              <a
                class="forgot-label"
                href="/signup"
              >
                Signup
              </a>
            </div>
          </form>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
